driver-core/src/main/com/mongodb/internal/connection/ProtocolHelper.java

```diff
@@ -63,13 +59,6 @@
     private static final Logger PROTOCOL_EVENT_LOGGER = Loggers.getLogger("protocol.event");
     private static final CodecRegistry REGISTRY = fromProviders(new BsonValueCodecProvider());
 
-    private static WriteConcernResult createWriteResult(final BsonDocument result) {
```

Comment:
I spotted this unused code when looking to see how these exceptions are used by the driver, and thought it should be removed so that it's clear that the exceptions are not actually used except where indicated in the new javadoc.